src/renderers/dom/shared/__tests__/ReactDOMComponent-test.js

```diff
@@ -198,6 +198,13 @@ describe('ReactDOMComponent', () => {
       );
     });
 
+    it('should not warn if property `on` is passed on a custom component', () => {
+      spyOn(console, 'error');
+      var container = document.createElement('div');
+      ReactDOM.render(<amp-image on="tap" />, container);
+      expectDev(console.error.calls.count()).toBe(0);
```

Comment:
You don't need this (or `spyOn` call). By default we assume none of the tests warn unless they opt in.

Reply:
Instead you'll want to assert that the `on` attribute gets set (which it probably doesn't right now).